USFMToolsSharp/Models/HTMLConfig.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace USFMToolsSharp.Models
+{
+    public class HTMLConfig
+    {
+        public List<string> divClasses;
+        public string encoding;
```

Comment:
Ok I missed this yesterday. Encoding is specified in the USFM so it shouldn't be passed in the config. Other then that we're all good.